server/src/main/scala/za/co/absa/atum/server/api/exception/AppError.scala

```diff
@@ -16,9 +16,6 @@
 
 package za.co.absa.atum.server.api.exception
 
-sealed trait AppError extends Throwable {
+abstract class AppError extends Throwable {
```

Comment:
About the first question. It's changed to abstract class because we now have a hierarchy or exceptions. Sealed traits are used for ServiceError subtypes and DatabaseError subtypes.